lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -413,6 +413,7 @@ def __init__(
         aryn_api_key: str = "",
         aryn_partitioner_address: str = DEFAULT_ARYN_PARTITIONER_ADDRESS,
         use_cache=False,
+        selection_size: int = 25,
```

Comment:
Why do you like `selection_size` better than `num_pages_per_call`? The latter sounds a bit clearer to me, though I can be convinced otherwise.

Reply:
Changed to `pages_per_call`.